sdk/hdinsight/ci.yml

```diff
@@ -29,6 +29,16 @@ pr:
       - sdk/hdinsight/pom.xml
       - sdk/hdinsight/azure-resourcemanager-hdinsight/pom.xml
 
+parameters:
+- name: release_azureresourcemanagerhdinsight
```

Comment:
These parameters is used to release the two packages seperately? If we don't add it, the two packages will be released together because they are both under `hdinsight` folder?

Reply:
yes

usually this is done when there is package of data-plane and mgmt-plane in same namespace. Now this is just multiple mgmt-plane.